src/utils/extension-utils.ts

```diff
@@ -24,6 +26,38 @@ export function identifierFromConfiguration(configuration: any) {
   return configuration.name.replace('@', '').replace('/', '.');
 }
 
+export async function fetchRemoteTypes(extensionRoot = process.cwd()) {
+  ux.action.start('Downloading JSON schemas and TypeScript types from Aha!');
+
+  // prettier-ignore
+  const typings = {
+    './types/aha-components.d.ts': 'https://cdn.aha.io/assets/extensions/types/aha-components.d.ts',
+    './types/aha-models.d.ts': 'https://cdn.aha.io/assets/extensions/types/aha-models.d.ts',
+    './schema/schema.json': 'https://cdn.aha.io/assets/extensions/schema/schema.json',
+    './schema/package-schema.json': 'https://cdn.aha.io/assets/extensions/schema/package-schema.json',
+  };
```

Comment:
Might seem a bit verbose, but it felt safer than doing any magic where I assume the local and remove URIs have the same suffix.